core-backend/src/funcs.rs

```diff
@@ -34,7 +34,7 @@ use blake2_rfc::blake2b::blake2b;
 use core::marker::PhantomData;
 use gear_core::{
     buffer::{RuntimeBuffer, RuntimeBufferSizeError},
-    costs::RuntimeCosts,
+    costs::CostToken,
```

Comment:
And I'd consider naming of just Cost mb

Reply:
But it's not `Cost`, it is just case for which we should calculate cost.